src/main/java/org/assertj/core/error/ShouldBeUnmodifiable.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.error;
+
+public class ShouldBeUnmodifiable extends BasicErrorMessageFactory {
+
+  private static final String SUCCESS_PATTERN = "%n" +
```

Comment:
maybe rename to `UNEXPECTED_SUCCESS_PATTERN` just because a success makes you think everything went fine